toolsets/src/gcc.cpp

```diff
@@ -49,7 +49,11 @@ std::string GccToolset::getLibNameFor(const Component &component)
 
 std::string GccToolset::getExeNameFor(const Component &component)
 {
+#if defined(_WIN32)
+    return getNameFor(component) + ".exe";
```

Comment:
Can you add an Appveyor config for this? I believe this is MingW, right?

Reply:
see pr #56